subsys/bootloader/cmake/sign.cmake

```diff
@@ -196,3 +196,24 @@ foreach (slot ${slots})
     )
 
 endforeach()
+
+if (CONFIG_BUILD_S1_VARIANT)
+
+  include(${NRF_DIR}/cmake/fw_zip.cmake)
+
+  # Generate zip file with both update candidates
+  set(s0_bin_path ${PROJECT_BINARY_DIR}/signed_by_b0_s0_image.bin)
+  get_filename_component(s0_name ${s0_bin_path} NAME)
```

Comment:
Does this just resolve to "signed_by_b0_s0_image"? In that case, I'd prefer

```suggestion
  set(s0_name signed_by_b0_s0_image)
  set(s0_bin_path ${PROJECT_BINARY_DIR}/${s0_name}.bin)
```

Reply:
Almost, the .bin is in the component, updated to that.